cli/src/main.rs

```diff
@@ -39,6 +39,8 @@ const CIRCUIT_CREATE_AFTER_HELP: &str = r"DETAILS:
     One or more nodes must be specified using the --node and/or --node-file arguments. These
```

Comment:
Could you rename `CIRCUIT_CREATE_AFTER_HELP` -> `CIRCUIT_PROPOSE_AFTER_HELP` in another commit while you're here? I missed that when I renamed the subcommand.